ch_backup/clickhouse/control.py

```diff
@@ -277,25 +277,14 @@ def __init__(self, ch_ctl_config: dict, main_config: dict) -> None:
         self._disks = self.get_disks()
         settings = {
             "allow_experimental_database_materialized_postgresql": 1,
+            "allow_experimental_database_materialized_mysql": 1,
             "allow_experimental_database_replicated": 1,
             "allow_experimental_funnel_functions": 1,
             "allow_experimental_live_view": 1,
+            "allow_experimental_window_view": 1,
             "allow_suspicious_codecs": 1,
             "allow_suspicious_low_cardinality_types": 1,
         }
-        if self.ch_version_ge("21.9"):
-            settings.update(
-                {
-                    "allow_experimental_database_materialized_mysql": 1,
-                    "allow_experimental_nlp_functions": 1,
```

Comment:
Enabling `allow_experimental_nlp_functions` leads to the following error in 23.9.

```
clickhouse01.test_net_1348 :) set allow_experimental_nlp_functions=1

clickhouse01.test_net_1348 :) SELECT name, create_query FROM system.functions WHERE origin == 'SQLUserDefined'

Received exception from server (version 23.9.1):
Code: 318. DB::Exception: Received from localhost:9000. DB::Exception: No lemmatizers specified in server config on prefix 'lemmatizers'. (INVALID_CONFIG_PARAMETER)
```